### DISCUSSION TITLE
Expand tabs button/ Auto-hide feature.

### DISCUSSION THREAD

**Discussion by PanajotMh** [piroor/treestyletab#3373] [Q&A]
When the pointer isn't hovered over the tree style tab it should automatically hide, when I hover the mouse the tree style tab should expand in its original size.  Or if possible a button similar to brave vertical tabs where you can hide/show by clicking.

**Comment by irvinm** [comment]
Are you talking about the sidebar itself contracting and expanding?  If so, addons can not do that themselves.  There are ways to emulate that behavior via CSS in userChrome.css, but those are all custom behaviors.  There are probably some examples mentioned in either the bugs list and\or discussions here, but the best place for this idea might be https://www.reddit.com/r/FirefoxCSS/.

Search:  https://www.google.com/search?q=firefox+auto+hide+sidebar+userChrome.css+site:www.reddit.com
